lib/tranzito_utils/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module TranzitoUtils
-  VERSION = "1.0.2"
+  VERSION = "1.1.0"
```

Comment:
Nit: this shouldn't be `1.0.3`?

Reply:
This is a breaking change, because it renamed a class. In reality, it should probably bump the major version. But - it isn't being used anywhere yet, so sure, let's just bump the patch.